linter workflowのエラーを修正する

```
Run npm run lint
  npm run lint
  shell: /usr/bin/bash -e {0}
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /home/runner/work/ssl_gui_test/ssl_gui_test/package.json
npm ERR! errno -[2](https://github.com/HansRobo/ssl_gui_test/actions/runs/10490239186/job/29056531345?pr=8#step:5:2)
npm ERR! enoent ENOENT: no such file or directory, open '/home/runner/work/ssl_gui_test/ssl_gui_test/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/202[4](https://github.com/HansRobo/ssl_gui_test/actions/runs/10490239186/job/29056531345?pr=8#step:5:5)-08-21T12_49_49_208Z-debug.log
Error: Process completed with exit code 2[5](https://github.com/HansRobo/ssl_gui_test/actions/runs/10490239186/job/29056531345?pr=8#step:5:6)4.
```

--- a/.github/workflows/lint.yml
+++ b/.github/workflows/lint.yml
@@ -22,7 +22,7 @@ jobs:
           node-version: '14'
 
       - name: Install dependencies
-        run: npm install
+        run: cd frontend && npm install
 
       - name: Run ESLint
-        run: npm run lint
+        run: cd frontend && npm run lint

